website/src/install.md

```diff
@@ -64,17 +56,28 @@ rpm -i git-town_7.6.0_linux_intel_64.rpm
 On Arch Linux, install the
 [git-town](https://aur.archlinux.org/packages/git-town) package from the AUR.
 
-You can also install Git Town via
+You can install Git Town via
 [Homebrew for Linux](https://docs.brew.sh/Homebrew-on-Linux):
 
 ```
 brew install git-town
 ```
 
-Manual installation:
-[download](https://github.com/git-town/git-town/releases/latest) the archive
-containing the Linux binary, extract it, and move the `git-town` binary into a
-directory listed in your`$PATH`.
+You can also [install Git Town manually](#manual-installation) or
+[compile from source](#compile-from-source).
+
+## manual installation
+
+```
+curl https://git-town.com/install.sh | sh
+```
+
+This puts the Git Town executable into `~/.local/bin`. Please add this directory
+to your PATH. For a completely custom installation:
+
+- [download](https://github.com/git-town/git-town/releases) the archive matching
+  your CPU architecture, extract it, and move the `git-town` executable into a
+  directory listed in your`$PATH`, for example`/usr/local/bin`.
```

Comment:
Thoughts on structuring this a little differently like the following

---

Run `curl https://git-town.com/install.sh | sh`

Alternatively [download](https://github.com/git-town/git-town/releases) the archive matching your CPU architecture, extract it, and move the `git-town` executable into a directory listed in your `$PATH`, for example`/usr/local/bin`.

Reply:
Great idea, implemented it. I keep the triple backticks because they give a button to copy the command to run. See https://www.git-town.com/install.html for reference.